packages/prisma/schema.prisma

```diff
@@ -1466,6 +1466,72 @@ view BookingTimeStatus {
   isTeamBooking  Boolean
 }
 
+model BookingDenormalized {
+  id             Int           @id @unique
+  uid            String
+  eventTypeId    Int
+  title          String?
+  description    String?
+  startTime      DateTime
+  endTime        DateTime
+  createdAt      DateTime
+  updatedAt      DateTime?
+  location       String?
+  paid           Boolean?
+  status         BookingStatus
+  rescheduled    Boolean?
+  userId         Int
+  teamId         Int?
+  eventLength    Int
+  eventParentId  Int?
+  userEmail      String
+  userName       String?
+  userUsername   String?
+  ratingFeedback String?
+  rating         Int?
+  noShowHost     Boolean?
+  isTeamBooking  Boolean
+
+  @@index([userId])
+  @@index([createdAt])
+  @@index([eventTypeId])
+  @@index([eventParentId])
+  @@index([teamId])
+  @@index([startTime])
+  @@index([endTime])
+  @@index([status])
+  @@index([teamId, isTeamBooking])
+  @@index([userId, isTeamBooking])
+}
+
+view BookingTimeStatusDenormalized {
+  id             Int           @id @unique
+  uid            String
+  eventTypeId    Int
```

Comment:
I see that some fields that are optional in BookingTimeStatus are required here. e.g., uid and eventTypeId

Intended, right?